### PR TITLE
Add attr ‘clearOnDoubleClick’,double click to clear pad, this feature is disabled by default

### PR DESCRIPTION
**Pull request by xiaoyaolml**
usage:
```xml
<com.github.gcacace.signaturepad.views.SignaturePad  
                android:layout_width="fill_parent"  
                android:layout_height="fill_parent"  
                android:id="@+id/signature_pad"  
                signature:clearOnDoubleClick="true"/>
```

### CODE REVIEW

**Comment by gcacace** [comment]
Can you please extend `README.md` with the new feature and config details?

File: signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java [hunk 1]
```diff
@@ -333,6 +338,25 @@ public Bitmap getTransparentSignatureBitmap(boolean trimBlankSpace) {
       return Bitmap.createBitmap(mSignatureBitmap, xMin, yMin, xMax - xMin, yMax - yMin);
     }
 
+    private boolean isDoubleClick() {
+        if (mClearOnDoubleClick) {
+            if (mFirstClick != 0 && System.currentTimeMillis() - mFirstClick > 200) {
```
Comment (gcacace) [inline signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java]: Could you extract the millisecond `200`, value into a `private static final int DOUBLE_CLICK_DELAY = 200;`?

File: signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java [hunk 2]
```diff
@@ -333,6 +338,25 @@ public Bitmap getTransparentSignatureBitmap(boolean trimBlankSpace) {
...
+                mCountClick = 0;
+            }
+            mCountClick++;
+            if (mCountClick == 1) {
+                mFirstClick = System.currentTimeMillis();
+            } else if (mCountClick == 2) {
+                long lastClick = System.currentTimeMillis();
+                if (lastClick - mFirstClick < 200) {
```
Comment (gcacace) [inline signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java]: As per my previous comment, please extract that milliseconds value.

Comment (xiaoyaolml) [inline signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java]: OK.

**Pull request merged**